menus/open-sesame.cson

```diff
@@ -11,3 +11,8 @@
     ]
   }
 ]
+
+'context-menu':
+  'atom-pane[data-active-item-path] .item-views': [
```

Comment:
Also this functionality works a _little_ odd IMO because if your cursor is on line 3 for instance and you right click line 2, the text cursor doesn't move so you're still opening the file from line 3 even though you clicked line 2. I'm sure there's a way around that but it is a little strange right now. Any ideas on how to work around that easily?